diagnostic.md

```diff
@@ -6,21 +6,22 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    The router is to set up the urls to listen to and construct the aproprite patttenr to listen to. Routes and sub routes
+    The main task is `route` is retrive data from the model and prepit to be used in the template.
```

Comment:
Yes, the Ember Application Router defines the route, and the Ember Route specifies how to load model data using the model hook.